doc/POINTS_COORDINATES.md

```diff
@@ -0,0 +1,207 @@
+# Points, tripoints, and coordinate systems
+
+## Axes
+
+The game is three-dimensional, with the axes oriented as follows:
+* The **x-axis** goes from left to right across the display (in non-isometric
+  views).
```

Comment:
Is it worth specifying that -X is to the left/west and +X is to the right/east or have we held people's hands here enough?